libsplinter/Cargo.toml

```diff
@@ -54,7 +54,7 @@ log = "0.4"
 metrics = {version = "0.17", features = ["std"], optional = true}
 mio = { version = "0.6", default-features = false }
 mio-extras = "2"
-oauth2 = { version = "3.0", optional = true }
+oauth2 = { version = "4.1.0", optional = true }
```

Comment:
Set this to "4" unless the intent is to explicitly say that it will only work with 4.1 or later; in that case, set it to "4.1".